deepchem/models/jax_models/layers.py

```diff
@@ -99,3 +103,71 @@ def __call__(self, inputs: jnp.ndarray) -> jnp.ndarray:
       output += bias
 
     return output
+
+
+def layer_norm(arr: jnp.ndarray, name: Optional[str] = None) -> jnp.ndarray:
+  """
+  Implementation of LayerNorm
+  
+  Parameters
+  ----------
+  
+  Inputs:
```

Comment:
The formatting here doesn't match numpy formatting style. It might be useful to check other docstrings to verify the formatting